Added some fixes for a couple of bugs I spotted in runner

boxadesign opened the pull request:
### What is the context of this PR?
The latest version of PL had a couple of bugs only spotted when used by runner.

This PR fixes those bugs so we need to add this to the latest release or create new one.

### How to review 
Check the code.

armstrongb commented:
@boxadesign do you have screenshots of what errors these fix? And or details of what browsers this was applicable to?

boxadesign commented:
@armstrongb The work I did was specifically on the repeating answers section. The bug  details and fixes are as follows:

- Add a class to allow buttons to maintain their width on mobile instead of going full width. This was applicable on the "Remove" button on a repeating answer. It came about from the button link style being removed and replaced with the small secondary which didn't look right being 100% width on mobile.

- Added a width to the repeating answers block so that when we float the "remove" button to the right it falls flush with parent content. Before adding the fix it floated flush with the main container.

- The details component required overflow hidden on the content element inside.

armstrongb commented:
LGTM, will merge
